content/using-atmosphere-packages.md

```diff
@@ -0,0 +1,65 @@
+---
+title: Using Atmosphere Packages
+order: 28
+discourseTopicId: 20192
+---
+
+Atmosphere packages are packages written specifically for Meteor and have several advantages over npm when used with Meteor. In particular, Atmosphere packages can:
```

Comment:
I'd move this to the atmosphere vs npm section, since if you are comparing the two you would want this information.